code/obj/machinery/deep_fryer.dm

```diff
@@ -234,10 +234,8 @@
 			fryholder.amount = 5
 		else
 			fryholder.amount = src.fryitem.w_class
-		fryholder.reagents = src.fryitem.reagents
-		if(src.cooktime >= 60)
-			fryholder.reagents.maximum_volume += 25
-			fryholder.reagents.add_reagent("friedessence",25)
```

Comment:
Where did this go?

Reply:
It exists up above as code that used to not work but now does

Reply:
Ooh, so that's why [READACTED] was broken originally.